randovania/games/am2r/logic_database/Industrial Complex.txt

```diff
@@ -219,6 +219,8 @@ Extra - minimap_data: [{'x': 58, 'y': 20}, {'x': 58, 'y': 21}, {'x': 58, 'y': 22
           Speed Booster and Shinesparking Tricks (Beginner)
           # IBJ
           Infinite Bomb Jumping (Expert) and Can Use Bombs
+  > Above torizo
+      Hi-Jump Boots and Morph Ball and Morph Glide (Intermediate)
```

Comment:
Can this be done without HJ? Assume yes at a higher morph glide level?
Doesn't matter much to me for inclusion because at that point I'm sure people just WJ from below anyways.

Reply:
idk maybe? pretty sure that would require walljumps

Reply:
Oh, is it gliding from above the torizo room (where the gravitts are) to the spike wall on the right?

Reply:
![grafik](https://github.com/randovania/randovania/assets/38186597/1d029d84-f341-4f2f-afb1-6b1dbfaf531c)

Reply:
If you want to, sure

Reply:
Doable with only just morph, not even power grip needed. Posted vid in the dev channel since it's too big apparently